### ISSUE TITLE
Add the option to force close

### ISSUE THREAD

**Issue by JavierRSobrino** [shesek/spark-wallet#97]
In the channels view, currently the option of forcing close a channel is not present, so if you close a channel who has been offline for a long time, you are forced to go to use the lightning-cli command line in order to force an unilateral close.

Please add the option to force close a channel.

**Comment by JavierRSobrino** [comment]
Related issue in clightning here: https://github.com/ElementsProject/lightning/issues/2791

**Comment by shesek** [comment]
Thanks for the suggestion. I'll wait for a bit to see what they settle on in https://github.com/ElementsProject/lightning/issues/2791 before changing this here.

On a semi related note, are you aware of the rpc console available from within spark itself? You can enable this by clicking the version number on the bottom left, which enables developer mode, opens up a couple new menu items on the main screen and adds some debugging information in various places.

**Comment by JavierRSobrino** [comment]
Yes, thanks, I knew I can launch lightning-cli commands in advanced options, that's what I usually do if the command is not very complex. Still this is not optimal and I think it should be used only in case of emergency or advanced tweaks.
